app/src/main/java/in/co/rajkumaar/amritarepo/timings/PublicTransportsActivity.java

```diff
@@ -433,5 +643,55 @@ public void onClick(DialogInterface dialogInterface, int i) {
         }
         return super.onOptionsItemSelected(item);
     }
+    private void countdown(long timeDiff)
```

Comment:
Whenever you commit files, ensure you `Reformat Code` to maintain code standards and good indendation.